src/CallTrait.php

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-namespace Google\Cloud\Vision\Annotation;
+namespace Google\Cloud;
```

Comment:
Maybe we should think about adding some sort of `Etc/` folder or something under `src/` to hold this and other helper files?

There are quite a few files in there now, and I've got another trait coming with Datastore as well.